admin/class-social-facebook-form.php

```diff
@@ -141,7 +141,7 @@ private function manage_user_admin() {
 		$nonce       = false;
 		$class_attr  = ' class="hidden"';
 
-		if ( is_array( $this->options['fb_admins'] ) && $this->options['fb_admins'] !== array() ) {
+		if ( is_array( $this->fb_admins ) && $this->fb_admins !== array() ) {
```

Comment:
The logic of this is exactly the same as the method `get_clearall`. We might consider renaming that method to make it more reusable

Reply:
Will fix this in a new commit.